### ISSUE TITLE
[feature request] ability to create already deleted file

### ISSUE THREAD

**Issue by shantanugadgil** [sl1pm4t/terraform-provider-lxd#66]
During my experiments, I forgot to tune down `parallelism` to a good working value, and ended up with a mixed-up state.

The file "data.cfg" never got created the first time round.

To make TF recreate the file, I changed the order of the file blocks inside the `lxd_container`.
It detects the change alright, but errors out with the following

```
Error applying plan:
1 error(s) occurred:
* lxd_container.test1: 1 error(s) occurred:
* lxd_container.test1: Could not delete file /etc/file/path/data.cfg: not found
```

Would there be any way to taint the `file` block inside the resource (I am not sure there is a way).

OR

Can there be an extra option inside the `file` block (or `lxd_container`) to force re-create files which are not found in the target container.

Regards,
Shantanu

**Comment by jtopjian** [comment]
Both options might be possible. I'll look into them this week.

**Comment by jtopjian** [comment]
Just a quick update: I'm going to wait until #64 is resolved before working on this to prevent too many modifications in one area at the same time.

**Comment by shantanugadgil** [comment]
Thanks @jtopjian 
@sl1pm4t  Any chances of a new release build from master after merging the few PRs still not in master?

Thanks and Regards,
Shantanu

**Comment by jtopjian** [comment]
This should be fixed in #77 

If an error occurs during creation before the file was uploaded, the file should be uploaded upon doing another apply.
